### ISSUE TITLE
[BUG] Broken Audio (Music/SFX)

### ISSUE THREAD

**Issue by neolightning** [MBU-Team/OpenMBU#106]
Music seems to be pitching wrong and sound effects are breaking.

Marble rolling sounded like chalkboard at some point after it was playing at a insanely high pitch.

**Comment by thearst3rd** [comment]
Just documenting what we've discussed in the past few days.

We've discovered that for some people, the game uses DirectSound, but for others, it uses OpenAL. The audio issues seemingly only happen when the game is using DirectSound.

Before this was known, I made pull request #98 which attempted to fix the pitch errors. It works for the most part, but there are still instances where the pitch of the marble can get incorrectly changed. One easy way to reproduce it is by repeatedly using super speeds to get ridiculous speeds in the half pipe in Skate Battle Royal, demonstrated here: https://www.youtube.com/watch?v=AaIZrUWdDL4

Two options:
1. Figure out why sometimes DirectSound is used instead of OpenAL and ensure that OpenAL is _always_ used. If we can do this, then we could also consider removing DirectSound completely from OpenMBU.
2. Figure out the bug with DirectSound and ensure that the pitch is always correct, so that both OpenAL and DirectSound are supported.

With either of those proper solutions, we should also then revert #98.
